docs/source/sample_datasets.rst

```diff
@@ -30,22 +30,48 @@ Creating random data
 
 Random data can be generated in the following way. It should be kept in mind
 that the data generated in this way can only be used for simple testing as it
-has no physical significance.
+has no physical significance. You can then load the data through the
+:ref:`api documentation`.
```

Comment:
This should link directly to the ["Loading data"](https://libertem.github.io/LiberTEM/formats.html) section, I think.